src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-details/rbd-details.component.html

```diff
@@ -149,6 +149,7 @@
          i18n>Performance</a>
       <ng-template ngbNavContent>
         <cd-grafana [grafanaPath]="rbdDashboardUrl"
+                    [datasource]="'Dashboard1'"
```

Comment:
What about for perf metrics?:
```suggestion
                    [type]="'metrics'"
```
And for logs:
```suggestion
                    [type]="'logs'"
```

Reply:
Done.